src/Components/Recommendation/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { useIntl } from 'react-intl';
+
+import routerParams from '@redhat-cloud-services/frontend-components-utilities/RouterParams';
+
+import { Recommendation } from './Recommendation';
+import { useGetRuleByIdQuery } from '../../Services/SmartProxy';
+import { getPluginName } from '../../Utilities/RuleName';
+
+const RecommendationWrapper = routerParams(({ match }) => {
+  const intl = useIntl();
```

Comment:
Same thing as before, prefer invocation of intl where its being used rather than passing the basket around